tlsfuzzer/messages.py

```diff
@@ -602,7 +608,8 @@ def generate(self, status):
             if cert_req is not None:
                 self.msg_alg = next((sig for sig in
                                      cert_req.supported_signature_algs
-                                     if sig[1] == SignatureAlgorithm.rsa),
+                                     if sig[1] == SignatureAlgorithm.rsa or
```

Comment:
That part seems to cause the following error:

test-rsa-pss-sigs-on-certificate-verify.py:stdout:  File "scripts/test-rsa-pss-sigs-on-certificate-verify.py", line 442, in main
test-rsa-pss-sigs-on-certificate-verify.py:stdout:    runner.run()
test-rsa-pss-sigs-on-certificate-verify.py:stdout:  File "/home/nmavrogi/cvs/gnutls/tests/suite/tls-fuzzer/tlsfuzzer/tlsfuzzer/runner.py", line 169, in run
test-rsa-pss-sigs-on-certificate-verify.py:stdout:    node.process(self.state, msg)
test-rsa-pss-sigs-on-certificate-verify.py:stdout:  File "/home/nmavrogi/cvs/gnutls/tests/suite/tls-fuzzer/tlsfuzzer/tlsfuzzer/expect.py", line 405, in process
test-rsa-pss-sigs-on-certificate-verify.py:stdout:    .format(cert_request.supported_signature_algs)
test-rsa-pss-sigs-on-certificate-verify.py:stdout:AssertionError: Unexpected algorithms found: [(4, 1), (4, 3), (5, 1), (5, 3), (6, 1), (6, 3), (3, 1), (3, 3), (2, 1), (2, 3), (8, 4), (8, 5), (8, 6), (4, 2), ...]

What is the intention of that test?